src/hard.js

```diff
@@ -0,0 +1,90 @@
+function fizzBuzz(start, end) {
+  let fizzBuzz = [];
+
+  for (let i = start; i < end; i++) {
+    if (i % 15 === 0) {
+      fizzBuzz.push("fizzbuzz");
+    } else if (i % 3 === 0) {
+      fizzBuzz.push("fizz");
+    } else if (i % 5 === 0) {
+      fizzBuzz.push("buzz");
+    } else {
+      fizzBuzz.push(i);
+    }
+  }
+  return fizzBuzz;
+}
+
+function hasDuplicate(numbers) {
+  let numberTally = {};
+
+  for (let number = 0; number < numbers.length; number++) {
```

Comment:
the name `number` makes this kind of hard to read with the given input `numbers`

we can just convert this into a for-of loop instead

Reply:
made the for of loop, Do I still need to change number to i?

Reply:
no in this case with the for of loop, we are using the actual value and not an index so `number` is good